File: takkernel/engine/src/main/java/com/atakmap/map/layer/raster/mobac/MobacMapTile.java

package com.atakmap.map.layer.raster.mobac;

import android.graphics.Bitmap;

public class MobacMapTile {

    public final Bitmap bitmap;
    public final byte[] data;
    public final long expiration;

    public MobacMapTile(Bitmap bitmap, byte[] data, long expiration) {
        this.bitmap = bitmap;
        this.data = data;
        this.expiration = expiration;
    }
}

File: takkernel/engine/src/main/cpp/raster/mobac/MobacMapTile.cpp
#include "raster/mobac/MobacMapTile.h"

using namespace atakmap::raster::mobac;


MobacMapTile::MobacMapTile() :
    data(NULL),
    dataLength(0),
    expiration(-1),
    releaseData(nullptr)
{
    bitmap.data = NULL;
    bitmap.dataLen = 0;
    bitmap.releaseData = NULL;
}
